client/k8s/application/Application.go

```diff
@@ -253,3 +262,16 @@ func ServerResourceForGroupVersionKind(discoveryClient discovery.DiscoveryInterf
 	}
 	return nil, errors.NewNotFound(schema.GroupResource{Group: gvk.Group, Resource: gvk.Kind}, "")
 }
+
+func (impl K8sClientServiceImpl) getNonNamespacedResourceKind(discoveryClient *discovery.DiscoveryClient) []string {
```

Comment:
We are returning only kinds, 
we should return gvks and check gvk (not only kind)

Reply:
I can't see changes in ListEvents and GetPodLogs, is this intentional?

Reply:
done, logical changes applied